Lib/test/test_source_encoding.py

```diff
@@ -14,11 +14,11 @@ class MiscSourceEncodingTest(unittest.TestCase):
 
     def test_pep263(self):
         self.assertEqual(
-            "�����".encode("utf-8"),
+            "ðÉÔÏÎ".encode("utf-8"),
```

Comment:
Sorry, these were changed unintentionally by my editor.  Going to revert...

Reply:
Ah, actually these were added by Github's generated commit [6d43cc](https://github.com/python/cpython/pull/96270/commits/6d43cc4e64cc02e561d8d4d1a1431fc316d52cd5) which accepted @ezio-melotti's suggestion.  Seems like a bug in Github, which isn't surprising, given this file is not valid UTF-8.  I'll clean this up by hand.